config/locales/pt-BR.yml

```diff
@@ -47,6 +47,7 @@ pt-BR:
       edit_post: Editar Post
       edit_private_topic: :thredded.nav.edit_topic
       edit_topic: Editar
+      mark_all_read: Marcar Como Lido
```

Comment:
Haaaaaa! Good effort! ![](http://img07.deviantart.net/e574/i/2004/08/0/6/brazilian_flag_emoticon_.gif) 🎉 

@ramirovjr what do you think about this one?